commons/clever-tools/ssh-access.md

```diff
@@ -45,6 +45,9 @@ If multiple instances are running, you will be asked which one to use:
     > Your choice: 1
     > Opening an ssh shell
     > bas@11281f38-31ff-43a7-8595-a2d82630c32b ~ $
+    
+###For Windows users
```

Comment:
Maybe a space after the `###` ? That's just cosmetic though

Reply:
I approve that. Let's roll it this way.